build-system/tasks/serve.js

```diff
@@ -30,29 +30,29 @@ const {
 } = require('../server/lazy-build');
 const {createCtrlcHandler} = require('../common/ctrlcHandler');
 const {cyan, green} = require('ansi-colors');
-const {getServeMode} = require('../server/app-utils');
+const {isRtvMode, getServeMode, setServeMode} = require('../server/app-utils');
 
-// Used for logging during server start / stop.
-let url = '';
+const argv = minimist(process.argv.slice(2), {string: ['rtv']});
+
+// Used for logging.
+let url = null;
+let {quiet} = argv;
 
 const serverFiles = globby.sync(['build-system/server/**']);
 
 /**
- * Logs the server's mode (based on command line arguments).
+ * Logs the server's mode.
  */
 function logServeMode() {
```

Comment:
What if we move this to `app-utils` so that you don't have to import `isRtvMode` or `getServeMode` just for this method?